tests/python_client/testcases/test_utility.py

```diff
@@ -1982,3 +1983,314 @@ def test_delete_user_root(self, host, port):
                                      password=ct.default_password, check_task=ct.CheckTasks.ccr)
         self.utility_wrap.delete_user(user=ct.default_user, check_task=ct.CheckTasks.err_res,
                                       check_items={ct.err_code: 31})
+
+    @pytest.mark.tags(CaseLabel.L3)
+    def test_clear_roles(self, host, port):
+        """
+        target: check get roles list and clear them
+        method: remove all roles except admin and public
+        expected: assert clear success
+        """
+        self.connection_wrap.connect(host=host, port=port, user=ct.default_user,
+                                     password=ct.default_password, check_task=ct.CheckTasks.ccr)
+        role_groups, _ = self.utility_wrap.list_roles(False)
+
+        for role_group in role_groups.groups:
+            if role_group.role_name not in ['admin', 'public']:
+                self.utility_wrap.init_role(role_group.role_name)
+                g_list, _ = self.utility_wrap.role_list_grants()
+                for g in g_list.groups:
+                    self.utility_wrap.role_revoke(g.object, g.object_name, g.privilege)
```

Comment:
role_revoke is necessary before role_drop? What will happen if we call role_drop directly?